src/ui/buttons/styles/responsive.js

```diff
@@ -1,29 +1,30 @@
 /* @flow */
 
 import { max, perc, roundUp } from 'belter/src';
-import { FUNDING } from '@paypal/sdk-constants/src';
+import { FUNDING, type LocaleType } from '@paypal/sdk-constants/src';
 
 import { BUTTON_SHAPE, BUTTON_LAYOUT, BUTTON_NUMBER, CLASS, ATTRIBUTE } from '../../../constants';
 import { BUTTON_SIZE_STYLE, BUTTON_RELATIVE_STYLE } from '../config';
 
+
 const BUTTON_MIN_ASPECT_RATIO = 2.2;
 const MIN_SPLIT_BUTTON_WIDTH = 300;
 
 const FIRST_BUTTON_PERC = 50;
 const WALLET_BUTTON_PERC = 60;
 
-export function buttonResponsiveStyle({ height } : {| height? : ?number |}) : string {
+export function buttonResponsiveStyle({ height, locale } : {| height? : ?number, locale? : ?LocaleType |}) : string {
 
     return Object.keys(BUTTON_SIZE_STYLE).map(size => {
 
         const style = BUTTON_SIZE_STYLE[size];
         const buttonHeight = height || style.defaultHeight;
         const minDualWidth = Math.max(Math.round(buttonHeight * BUTTON_MIN_ASPECT_RATIO * (100 / WALLET_BUTTON_PERC)), MIN_SPLIT_BUTTON_WIDTH);
 
-        const labelHeight = max(roundUp(perc(buttonHeight, 35) + 5, 2), 12);
+        const textPercPercentage = locale?.lang === 'de' ? 32 : 36;
```

Comment:
Should the default be `35` here to match previous code?

Reply:
Yes, missed it